supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -27,3 +27,29 @@ Provide the source language if it is supported by the Red Hat Customer Portal to
 == Long code examples in procedures
 
 All code blocks (regardless of length) must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts. The length of the block is irrelevant, within reason, if the code block follows these guidelines.
+
+[[commands-with-elevated-privileges]]
+== Commands with elevated privileges
+
+Some commands require elevated privileges to run. Users without elevated privileges must first do one of the following to run such a command:
+
+* Use `su -i` to switch to the root user account.
+* Preface the command with `sudo` to temporarily change their current privileges.
+
+=== Guidelines for documenting commands with elevated privileges
```

Comment:
Not critical, but I'd question whether this subheading is necessary, or if it can be removed. You say almost the same thing with the following sentence, and there aren't multiple subheadings, so I don't think this is necessary.

Reply:
+1